fehmtk/postprocessors/check_history.py

```diff
@@ -43,36 +43,69 @@ def check_history(
     if fields is None:
         fields = [field for field in history.columns if field not in ('time_days', 'node')]
 
-    plot_history(history, fields)
+    plot_history(history, fields, run_root=config.files_config.run_root)
     if interact:
         embed()
 
 
-def plot_history(history: pd.DataFrame, fields: list[str]):
+def plot_history(history: pd.DataFrame, fields: list[str], run_root=None):
     n_nodes = history.node.nunique()
+    marker = 'o' if history.time_days.nunique() < 75 else None
+    markersize = 4
 
     history = history.astype(float)  # Plotting doesn't play nice with Decimal types
     history['node'] = history.node.astype(int).astype(str)  # Remove trailing zeroes, treat as categorical
     history['time_years'] = history.time_days / 365
 
     sns.set_palette('muted')
     fig, axs = plt.subplots(len(fields), 1, figsize=(6, 2 + len(fields)), constrained_layout=True, sharex=True)
+    fig.suptitle(run_root or 'Run history')
     for i, (field, ax) in enumerate(zip(fields, axs)):
-        sns.lineplot(data=history, x='time_years', y=field, hue='node', ax=ax, legend=i == 0)
+        sns.lineplot(
+            data=history,
+            x='time_years',
+            y=field,
+            hue='node',
+            style='node',
+            ax=ax,
+            legend=i == 0,
+            marker=marker,
+            markersize=markersize,
+        )
         ax.set_xlabel(r'$t\:(years)$')
         ax.set_ylabel(PLOT_AXES_MAPPING.get(field, field))
 
     delta_history = history[['node', 'time_years']][n_nodes:]
     delta_history['delta_time_years'] = history.groupby('node').time_years.diff()
     for field in fields:
-        delta_history[field] = history.groupby('node')[field].diff()
+        delta_history[field] = 1000 * history.groupby('node')[field].diff() / delta_history.delta_time_years
```

Comment:
Please check this line, I want to be sure I have the logic right to get to per thousand years. I've divided by the actual time delta and multiplied the value by 1000.